.gitignore

```diff
@@ -0,0 +1 @@
+*.DS_Store
```

Comment:
Although I don't mind that it is here, please also add that to your global gitignore. There is never a case when you want that included in a repo.

```
git config --global core.excludesfile ~/.gitignore
echo "*.DS_Store" > ~/.gitignore
```
🥃